index.js

```diff
@@ -0,0 +1,30 @@
+const calculate = (arr) => {
+  const calculationData = sanitizeCalculationData(arr)
+  const n1 = calculationData[0]
+  const n2 = calculationData[2]
+  const operator = calculationData[1]
+
+  switch (operator) {
+    case '+': return n1 + n2
+    case '-': return n1 - n2
+    case '*': return n1 * n2
+    case '/': return n1 / n2
+    default: return NaN
+  }
+}
+
+const sanitizeCalculationData = (arr) => {
+  const forCalculation = []
+  const validOperators = ['+', '-', '*', '/']
+
+  for (let i = 0; i < arr.length; i++) {
+    if (validOperators.includes(arr[i])) { forCalculation.push(arr[i]) }
+    if (Number(arr[i])) { forCalculation.push(arr[i]) }
```

Comment:
You should be using `!isNaN()` here. `Number(0)` is `0` which is `false` so your function doesn't allow zeroes.